loki/batch/scheduler.py

```diff
@@ -503,9 +503,11 @@ def _get_definition_items(_item, sgraph_items):
 
         if transformation.creates_items:
             self._discover()
-
             self._parse_items()
 
+        if transformation.creates_edges:
+            self._discover()
```

Comment:
If no items need to be parsed, I don't expect calling `_parse_items` to be particularly expensive. Wouldn't re-using the `creates_items` manifest entry work here, too?

The reason is that I don't want to make the Transformation manifest unnecessarily complex.

Reply:
Yes `_parse_items` isn't too expensive; `_discover` and `_parse_items` both cost around ~14s for ecWAM. And yes, `creates_items` would also do the job and I am happy to just use that rather than adding this new option.